cmd/soroban-cli/src/commands/contract/fetch.rs

```diff
@@ -22,7 +22,10 @@ use crate::{
 pub struct Cmd {
     /// Contract ID to fetch
     #[arg(long = "id", env = "STELLAR_CONTRACT_ID")]
-    pub contract_id: config::UnresolvedContract,
+    pub contract_id: Option<config::UnresolvedContract>,
+    /// Wasm to fetch
+    #[arg(long = "wasm-hash", conflicts_with = "contract_id")]
```

Comment:
contract_id has an env var to it. Should we add one for wasm-hash as well? `STELLAR_WASM_HASH` maybe?

Reply:
I don't think we use a env var for the wasm hash on other commands, so I don't think so. Having an env var on the first option is also odd, but I don't think we need to remove it to create consistency without it.